microsoft-edge/puppeteer/index.md

```diff
@@ -3,109 +3,115 @@ description: Use Puppeteer to automate and test in Microsoft Edge.
+By default, when you install Puppeteer, the installer downloads a recent version of [Chromium](https://www.chromium.org/Home), the open-source browser that [Microsoft Edge is also built upon](https://blogs.windows.com/windowsexperience/2018/12/06/microsoft-edge-making-the-web-better-through-more-open-source-collaboration).
+
+If you have Microsoft Edge installed, you can use [puppeteer-core](https://pptr.dev/#?product=Puppeteer&version=v2.0.0&show=api-puppeteer-vs-puppeteer-core).  `puppeteer-core` is a lightweight version of Puppeteer that launches an existing browser installation, like Microsoft Edge.  To download Microsoft Edge, go to [Download Microsoft Edge Insider Channels](https://www.microsoftedgeinsider.com/download).
 
 
 <!-- ====================================================================== -->
 ## Installing puppeteer-core
 
-You may add `puppeteer-core` to your website or app with one of the following commands.
+You can add `puppeteer-core` to your website or app by using one of the following commands:
 
-```shell
+```console
 npm i puppeteer-core
 ```
 
-```shell
+```console
 yarn add puppeteer-core
 ```
 
 
 <!-- ====================================================================== -->
 ## Launch Microsoft Edge with puppeteer-core
 
-> [!NOTE]
-> `puppeteer-core` relies on Node v8.9.0 or later.  The example below uses `async`/`await` which is only supported in Node v7.6.0 or later.  Run `node -v` from the command-line to ensure you have a compatible version of Node.js.
+`puppeteer-core` is similar to other browser-testing-frameworks, such as [WebDriver](../webdriver-chromium/index.md).  You create an instance of the browser, open a webpage, and then manipulate the webpage by using the Puppeteer API.
 
-`puppeteer-core` should be familiar to users of other browser-testing-frameworks like [WebDriver](../webdriver-chromium/index.md).  You create an instance of the browser, open a page, and then manipulate it with the Puppeteer API.  In the following code sample, `puppeteer-core` launches Microsoft Edge, navigates to `https://www.microsoftedgeinsider.com`, and saves a screenshot as `example.png`.
+Microsoft Edge (EdgeHTML) doesn't work with `puppeteer-core`.  You must install the [Microsoft Edge Insider channels](https://www.microsoftedgeinsider.com/download) to follow this example.
```

Comment:
This is out of scope for this PR, but we should update this sentence. EdgeHTML-versions of Edge are no longer available, and there is no need to go the one of the insiders channels to get Chromium-versions of Edge.